.github/workflows/build-pr-image.yml

```diff
@@ -0,0 +1,97 @@
+
+name: Build PR Docker Image
+
+on:
+  push:
+    branches:
+      - "main"
+    paths-ignore:
+      - "**.md"
+  pull_request:
+    branches:
+      - "*"
+    paths-ignore:
+      - "**.md"
+env:
+    AWS_REGION: us-east-1
+
+jobs:
+    build:
+      name: build
+      permissions:
+        id-token: write
+        contents: read
+      runs-on: ubicloud-standard-16
```

Comment:
This should be able to run in the free `ubuntu-latest` runner.

Reply:
It took more than 25 minutes to release build (those are 2 cores + 7GB machines.)

Reply:
Fair, although it should come down to 12-15mins once dependencies are cache. The first run until cache is build always takes a long time 😂

If it stays slow then re-add the ubicloud runner